libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -542,7 +542,7 @@ const serializeParameterWithPath = (
     let p = result;
     while (pathSegments.length > 0) {
       const pathSegment = pathSegments.shift() as Segment;
-      const propertyKey = getAndEscapeSegment(pathSegment);
+      const propertyKey = getAndEscapeSegment(pathSegment, /** decodeSegment*/ false); // Disable decoding as it already happens with parseEx(valueKey) above
```

Comment:
Is there an issue with how we are double decoding and it is decoding an actual property name?